tests/webview/ui/test_version.py

```diff
@@ -0,0 +1,80 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from datetime import datetime, timedelta
+from warnings import warn
+from github3 import GitHub
+
+from tests import markers
+from tests.utils import shorten_tag
+
+from pages.webview.version import Version
+from pages.webview.history import History
+
+
+@markers.webview
+@markers.nondestructive
+def test_version(webview_base_url, selenium, record_property):
+    # GIVEN the webview base url, the Selenium driver and the time 2 days ago
+    two_days_ago = datetime.utcnow() - timedelta(days=2)
+
+    # WHEN the version and history pages have been visited
+    version = Version(selenium, webview_base_url).open()
+    version_parser = version.version_parser
+
+    history = History(selenium, webview_base_url).open()
+    history_url = selenium.current_url
+    release_parsers = history.release_parsers
+    current_release_parser = release_parsers[0]
+    current_version_parser = current_release_parser.version_parser
+
+    # THEN - the release is less than 2 days old (warning only)
+    #      - the tags for webview, archive and deploy are the latest tags (warning only)
+    #      - the last 2 releases in history.txt are not exactly identical (warning only)
+    #      - version.txt matches the JSON for the latest release in history.txt
+    if current_version_parser.datetime < two_days_ago:
+        warn('\n\nThe latest release in {url} ({date}) is more than 2 days old.'.format(
+                 url=history_url, date=current_version_parser.date))
+
+    tag_tests = [('webview', 'webview_tag'),
```

Comment:
Hmm do you want me to split this test into 2 then? So we don't have to visit the version and history pages 3 times each.

Reply:
yea, i had thought about that too. it seems as if that would make it better and separate concerns a bit more.

Reply:
it brought up the thought of, "what if there are two tests with the same test case id?" We may need to split it into two test cases in test rail too. if that is necessary you can ask @kerwinso to do that for you.

Reply:
Hmm that might not be an issue (maybe). The second part of the test never fails (it just prints a bunch of warnings related to the tags etc). So we could have the marker only on the first part (literally the only way for this test to fail is for things to be offline or if something weird happened during deployment and version.txt and history.txt don't match).

But if you think it would still be useful @kerwinso, please split test case C175150 (one part about ensuring version.txt and history.txt are present and match and one part about ensuring the release is less than 2 days old and contains the latest tags for webview, cnx-archive and cnx-deploy

Reply:
Actually, upon looking at this more this is OK. You're right splitting it up with paramatrize means we would have to hit those pages each time for each tag. This is more efficient. I would probably move the tags to the GIVEN section and add a comment that they are used to later to check each version against GitHub.